stages/01_download_data.py

```diff
@@ -0,0 +1,18 @@
+# Imports
+# -------
+import os
+import urllib.request
+
+os.makedirs('downloads', exist_ok=True)
```

Comment:
It's a small change, but use `download` instead of `downloads` to match up with other bricks.